src/main/java/frc/robot/ComponentsControl.java

```diff
@@ -13,9 +14,9 @@ public void runComponents(Components components, ControlInputs controlInputs, Se
         //Controls
             //Belts
         if (controlInputs.beltAuto) {
-            if (sensorInputs.currentYawDegrees <= -20.0 && sensorInputs.currentYawDegrees >= -160.0){
+            if (sensorInputs.currentYawDegrees <= -(90-beltGyroRotaionInsideDegreeToleranceFromY) && sensorInputs.currentYawDegrees >= -(90+beltGyroRotaionInsideDegreeToleranceFromY)) {
```

Comment:
Add spaces to function `(90-beltGyroRotaionInsideDegreeToleranceFromY)` and `(90+beltGyroRotaionInsideDegreeToleranceFromY)`